testscommon/cryptoMocks/multisignerMock.go

```diff
@@ -0,0 +1,237 @@
+package cryptoMocks
+
+import (
+	"github.com/ElrondNetwork/elrond-go-crypto"
+)
+
+const signatureSize = 48
+
+// MultisignerMock is used to mock the multisignature scheme
+type MultisignerMock struct {
+	aggSig      []byte
+	aggCom      []byte
+	commSecret  []byte
+	commHash    []byte
+	commitments [][]byte
+	sigs        [][]byte
+	pubkeys     []string
+	selfId      uint16
+
+	VerifyCalled                           func(msg []byte, bitmap []byte) error
```

Comment:
this is a combination of a mock and a stub. we have a separate stub, so please remove the handler functions and leave it as a mock only.

Reply:
I know, the thing is that the multisigner is a complex implementation underneath. The tests are written in such way that a complete refactoring will take a while. Will remain as it is for now.